src/main/audio/server/NonRealTimeAudioServer.hpp

```diff
@@ -73,6 +73,9 @@ namespace ctoot {
 			public:
 				// For compatibility with JUCE
 				void work(const float** inputBuffer, float** outputBuffer, int nFrames, int inputChannelCount, int outputChannelCount);
+				
+				//For compatibility with the PortAudio framework
+				void work(float* inputBuffer, float* outputBuffer, int nFrames, int inputChannelCount, int outputChannelCount);
```

Comment:
As is, this PR would not add an implementation for this signature. You had it correct in another PR. I.e. in `NonRealTimeAudioServer.cpp` you added:
```
void NonRealTimeAudioServer::work(float* inputBuffer, float* outputBuffer, int nFrames, int inputChannelCount, int outputChannelCount) {
	auto externalAudioServer = dynamic_pointer_cast<ExternalAudioServer>(server.lock());
	if (externalAudioServer) {
		externalAudioServer->work(inputBuffer, outputBuffer, nFrames, inputChannelCount, outputChannelCount);
	}
}
```